### PR TITLE
Pods/containers: expose resource requests & limits

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Expose container resource requests & limits on both `kube_pod_container` and `kube_pod` entities. This will allow us to more easily find problematic deployments.

### CODE REVIEW

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -220,6 +221,16 @@ def get_all(kube_client, kube_func, namespace=None, **kwargs) -> list:
     return items
 
 
+def parse_resources(resources):
```
Comment (mohabusama) [inline zmon_agent/discovery/kubernetes/cluster.py:224]: We are not handling the ``ValueError`` exception anywhere, and that would block the discovery of further resources - regardless of whether an invalid "limit" can make it to Kubernetes.

Comment (aermakov-zalando) [reply zmon_agent/discovery/kubernetes/cluster.py:224]: Good point, although I don't think it's possible to read invalid values out.

**Comment by aermakov-zalando** [comment]
Any updates here?

**Comment by Jan-M** [comment]
Good thing to have.

**Review by mohabusama — APPROVED** [review]

**Comment by mohabusama** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**